h2o-docs/src/product/performance-and-prediction.rst

```diff
@@ -2004,6 +2004,108 @@ Examples:
         # build the standardized coefficient magnitudes plot:
         glm.std_coef_plot()
 
+
+Gains/Lift 
+''''''''''
+
+Gains/Lift evaluates the prediction ability of a binary classification model. The chart is computed using the prediction probability and the true response labels. The accuracy of the classification model for a random sample is evaluated according to the results when the model is and is not used. The Gains/Lift chart shows the effectiveness of the current model(s) compared to a baseline, allowing users to quickly identify the most useful model. 
+
```

Comment:
Please change:

The accuracy of the classification model for a random sample is evaluated according to the results when the model is and is not used.

to

The accuracy of the baseline is evaluated when no model is used.  For instance, if there are x% positive responses in the dataset, when we grab 10% of the dataset, we can assume that there are 10% of the x% positive responses in the 10% dataset we chose.  If x = 10 and there are 10% positive responses in the dataset.  When we choose a 10% dataset, we will expect there to be 1% positive responses in the 10% dataset we chose.